spec/features/editing_file_attachments/upload_file_attachment_spec.rb

```diff
@@ -3,8 +3,21 @@
     given_there_is_an_edition
     when_i_go_to_edit_the_edition
     and_i_go_to_insert_an_attachment
+    and_i_select_a_file
     and_i_upload_a_file_attachment
     then_i_can_see_previews_of_the_attachment
+    and_i_can_see_a_link_to_the_attachment
+    and_i_see_the_timeline_entry
+  end
+
+  scenario do
```

Comment:
Can we add a description to the scenario to distinguish what they're doing? [Images are a good example of this.](https://github.com/alphagov/content-publisher/blob/master/spec/features/editing_images/edit_image_spec.rb)